docs/tutorial-modules/varjo.mdx

```diff
@@ -0,0 +1,76 @@
+# VRCFTVarjoModule (Varjo)
+
+## Summary
+
+The VRCFTVarjoModule contains the runtime used to interface with Varjo eye-tracking hardware and software on Windows PCs.
+
+## Set-up
+
+Setup of the Varjo Module is fairly simple, though can be a little confusing depending on your hardware/software configuration. Please read carefully to avoid any issues. 
+
+:::note
+This guide assumes you have the Varjo Base installed and updated prior to using the VRCFTVarjoModule. 
+If you do not have the [The Varjo Base](https://account.varjo.com/downloads#varjo_base)
+installed, please install it before continuing.
+:::
+
+### Installation
+
+To use the VRCFTVarjoModule, you will need a compatible headset and the Varjo Base installed.
+You can get a copy of the Varjo Base from Varjo's website, you will need an account to download it. 
+
+### Hardware
+
+The VRCFTVarjoModule is compatible with the following hardware:
+
+| Headset | Eye Tracking | Facial Tracking |
+| ---------------------------------- | -------- | --------- |
+| Varjo Aero                         | `✅` Yes | `❌` No  |
+| Varjo XR-3*                        | `✅` Yes | `❌` No  |
+| Varjo VR-3*						 | `✅` Yes | `❌` No  |
+
+*Untested
+
+## Using The Varjo Module
+
+To use your Varjo with VRChat, you will need to use one of two modules with the VRCFaceTracking software:
+
+- [M3ga's](https://github.com/m3gagluk/VRCFTVarjoModule) is considered to be the more "stable" option, but only has binary blinking (eyes are fully open and closed)
+- [Chickenbread's](https://github.com/Chickenbreadlp/VRCFTVarjoModule) is a experimental fork of M3ga's module. It supports variable eye openness available in newer Varjo SDK versions
+
+For more information on modules and how to use them, see [Modules](./modules.mdx).
+
+:::warning
+You MUST unblock the VRCFTVarjoModule.dll file after downloading it. This can be done by right-clicking the VRCFTVarjoModule.dll file, selecting "Properties", and then selecting the "Unblock" button.
+If you do not see this option, you do not need to worry about this step.
+:::
+<div style={{
+    width: '50%',
+    height: 'auto',
+    margin: 'auto',
+    display: 'block'
+}}>
+    <img src={require("./img/varjo/varjo_unblock_instructions.png").default} alt="varjo unblock instructions" />
+</div>
+
+Assuming there are no connection issues with the hardware, the connection hardware components should be initialized by VRCFT after a few seconds.
+
+### Calibrating Eye Tracking
+
+Eye tracking is calibrated automatically when you put on the headset, as a quick 1-point calibration.
+Additionally, you can perform a 5-point calibration from the leftmost tab in the Varjo Base.
+<div style={{
+    width: '50%',
+    height: 'auto',
+    margin: 'auto',
+    display: 'block'
+}}>
+    <img src={require("./img/varjo/varjo_unblock_instructions.png").default} alt="varjo unblock instructions" />
```

Comment:
Why are we using the same unblock image here for calibration?

Reply:
Resolved in #8c4eb085. This was only relevant to older versions of the VRCFaceTracking module